firmware/tunerstudio/rusefi.input

```diff
@@ -1480,41 +1480,35 @@ cmd_stop_engine              = "w\x00\x99\x00\x00"
 		panel = oilp_settings
 		gauge = OilPressGauge
 
-;			Engine->MAP Sensor Settings
-	dialog = mapSensorVDlg, "Voltage Sensor Settings", xAxis		
-	    field = "MAP ADC input", 						map_sensor_hwChannel, {hasFrequencyReportingMapSensor == 0}
-		field = "Map type", map_sensor_type, 			{hasFrequencyReportingMapSensor == 0}
+;			Sensors->MAP Sensor Settings
+	dialog = mapSensorVDlg, "Voltage Sensor Settings", yAxis		
+		field = "MAP type", map_sensor_type, 			{hasFrequencyReportingMapSensor == 0}
 		field = "MAP value low point",  		        map_sensor_lowValue, { map_sensor_type == 0 && hasFrequencyReportingMapSensor == 0}
 		field = "MAP voltage low point", 	 		    mapLowValueVoltage, { map_sensor_type == 0 && hasFrequencyReportingMapSensor == 0}
 		field = "MAP value high point",               	map_sensor_highValue, { map_sensor_type == 0 && hasFrequencyReportingMapSensor == 0}
 		field = "MAP voltage high value",               mapHighValueVoltage, { map_sensor_type == 0 && hasFrequencyReportingMapSensor == 0}
 		field = "Use minimal value of",					mapMinBufferLength
 		field = "Measure Map Only In One Cylinder",		measureMapOnlyInOneCylinder
 
-	dialog = mapSensorFDlg, "Frequency Sensor Settings", xAxis		
-		field = "pin",		frequencyReportingMapInputPin, {hasFrequencyReportingMapSensor == 1}
+	dialog = mapSensorFDlg, "Frequency Sensor Settings", yAxis
 		field = "0 kPa freq",		mapFrequency0Kpa, {hasFrequencyReportingMapSensor == 1}
 		field = "100 kpa freq",		mapFrequency100Kpa, {hasFrequencyReportingMapSensor == 1}
 
-	dialog = mapCurves, "", border		
-	    panel = map_samplingAngleCurve,	West
-		panel = map_samplingWindowCurve,	East
-
 	dialog = mapSwitch, "MAP input type"
 	    field = "frequency-based MAP",				hasFrequencyReportingMapSensor
 	    
-	dialog = mapVandF, "V and F"	    
-		panel = mapSensorVDlg, 		North
-		panel = mapSensorFDlg,		South
+	dialog = mapSwitchAndSettings, "", yAxis
+		panel = mapSwitch
```

Comment:
This is almost the same as 'mapVandF', but with 'yAxis' alignment and without 'North' and 'South'